lib/aurora_bootstrapper/utility.rb

```diff
@@ -0,0 +1,48 @@
+module AuroraBootstrapper
+  class Utility
+    def self.get_db_pass
+      db_pass = nil
+
+      if ENV.key?('DB_PASS')
+        # read from ENV directly
+        db_pass = ENV.fetch( 'DB_PASS')
+      else
+        file_object = File.open(ENV.fetch( 'DB_CRED_FILE' ))
+        # read from file stored in e.g. Vault etc
+        db_pass = file_object.read
+      end
+
+      db_pass
+    end
+
+    def self.get_db_user
```

Comment:
because ruby doesn't require parentheses, the ruby naming style discourages `get_` prefixes and encourages a naming convention that is descriptive of the return, like `Utility.db_pass`